hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/MergeOnReadIncrementalRelation.scala

```diff
@@ -94,9 +94,10 @@ case class MergeOnReadIncrementalRelation(override val sqlContext: SQLContext,
       tableState = tableState,
       mergeType = mergeType,
       fileSplits = fileSplits,
-      includeStartTime = includeStartTime,
+      includeStartTime = false,
```

Comment:
If `includeStartTime` is not used, should we get rid of it?

Reply:
Yes, we should get rid of it. Same applies to `startTimestamp`, and `endTimestamp`. I'll file a jira to track this